trunk/src/lsc/util.py

```diff
@@ -138,6 +138,31 @@ def readkey3(hdr,keyword):
     except: 
        _instrume='none'
     if 'kb' in _instrume: # SBIG
```

Comment:
Can this just be `if 'kb' in _instrume or 'sq' in _instrume:`? Pre-2014, different instruments had different header keywords, but they have been standard since then.

Reply:
fixed with recent push